### ISSUE TITLE
Surppress unity event invokes while game paused

### ISSUE THREAD

**Issue by Eddio0141** [Eddio0141/UniTAS#184]
Stuff like [Camera](https://docs.unity3d.com/ScriptReference/Camera.html).onPreCull is called even if mono behaviour is paused

**Comment by SebastianWozniczka** [comment]
Good morning, after 5 minutes I found an answer on another site that works a bit: 
[](https://answers.unity.com/questions/9485/is-there-any-way-to-disable-an-onprecull-function.html)

**Comment by SebastianWozniczka** [comment]
Good morning, after 5 minutes I found an answer on another site that works a bit:
https://answers.unity.com/questions/9485/is-there-any-way-to-disable-an-onprecull-function.html

**Comment by Eddio0141** [comment]
Yeah this could be a way to do but not sure if it'll work for all events I want to be stopped from being called.
This issue I made is talking more about stopping unity events called when I set `MonoBehaviourController.PausedExecution` to true with the camera class being an example.
There's few other classes I need to do this for.

I think I could temporarily remove all registered functions to somewhere else until I unpause
